### ISSUE TITLE
Question about the lidar data in OpenLane

### ISSUE THREAD

**Issue by rookiecsq** [JMoonr/mmlane#2]
Hi, thanks for your nice paper. I believe the method you proposed would be helpful for the 3d lane detection task. Since you have mentioned in the paper that there are corresponding lidar data in the OpenLane dataset, I wonder how to get them. 

![image](https://user-images.githubusercontent.com/52888992/223918243-3b65da9d-57f6-40cd-9a88-ae0cbdd2dee5.png)
I have checked the issues of the OpenLane repo, (https://github.com/OpenDriveLab/OpenLane/issues/37) and it seems they don't provide the corresponding lidar data. Does it mean that I have to access them in the original dataset Waymo, or there is a version of the OpenLane that already contain the corresponding lidar data?

**Comment by JMoonr** [comment]
Yes, we get the LiDAR data from the original Waymo Dataset. You can refer to Waymo to get them :)

**Comment by JMoonr** [comment]
Thanks. Yes, we get the LiDAR data from the original Waymo Dataset. You can refer to Waymo to get them :)

**Comment by rookiecsq** [comment]
ok, thanks! ^_^
